lib/vmpooler/pool_manager.rb

```diff
@@ -443,6 +537,15 @@ def execute!
       $redis.set('vmpooler__tasks__clone', 0)
 
       loop do
+        if ! $threads['snapshot_manager']
```

Comment:
It was to match the `! $threads[pool['name']]` check at line 550 below.  I can modify both if you'd prefer `unless`.